README.md

```diff
@@ -163,17 +168,21 @@ tracks don't.
 
 Some problems that are implemented in multiple tracks use the same inputs and
 outputs to define the test suites. Where the [x-common](https://github.com/exercism/x-common)
-repository contains json files with these inputs and outputs, we can generate
-the test cases programmatically.
+repository contains a *canonical-data.json* file with these inputs and outputs,
+we can generate the test cases programmatically.
 
-See the *example_gen.go* file in the `leap` exercise for an example of how this
+See the *gen.go* file in the `leap` exercise for an example of how this
 can be done.
 
+Test case generators are named *gen.go* and are kept in a special *.meta*
+directory within each exercise that makes use of a test cases generator. This
+*.meta* directory will be ignored when a user fetches an exercise.
+
 Whenever the shared JSON data changes, the test cases will need to be regenerated.
 To do this, make sure that the **x-common** repository has been cloned in the same
 parent-directory as the **xgo** repository. Then navigate into the **xgo**
-directory and run `go run exercises/<problem>/example_gen.go`. You should see
-that the `<problem>/test_cases.go` file has changed. Commit the change.
+directory and run `go run exercises/<exercise>/.meta/gen.go`. You should see
+that the `<exercise>/test_cases.go` file has changed. Commit the change.
```

Comment:
funnily enough, it's `cases_test.go` (simply so that it has the `_test.go` suffix), so while you're changing this line from problem->exercise, probably good to correct this too?